services/database/vector/threats.yaml

```diff
@@ -0,0 +1,129 @@
+shared-threats:
+  - reference-id: CCC
+    identifiers:
+      - CCC.TH01 # Access Control is Misconfigured
+      - CCC.TH02 # Data is Intercepted in Transit
+      - CCC.TH03 # Deployment Region Network is Untrusted
+      - CCC.TH04 # Data is Replicated to Untrusted or External Locations
+      - CCC.TH05 # Data is Corrupted During Replication
+      - CCC.TH06 # Data is Lost or Corrupted
+      - CCC.TH07 # Logs are Tampered With or Deleted
+      - CCC.TH08 # Cost Management Data is Manipulated
+      - CCC.TH09 # Logs or Monitoring Data are Read by Unauthorized Users
+      - CCC.TH10 # Alerts are Intercepted
+      - CCC.TH11 # Event Notifications are Incorrectly Triggered
+      - CCC.TH12 # Resource Constraints are Exhausted
+      - CCC.TH13 # Resource Tags are Manipulated
+      - CCC.TH14 # Older Resource Versions are Exploited
+      - CCC.TH15 # Automated Enumeration and Reconnaissance by Non-human Entities
+      - CCC.TH16 # Logging and Monitoring are Disabled
+      - CCC.TH17 # Unauthorized Network Access via Misconfigured Rules
+threats:
+  - id: CCC.Vector.TH01
+    title: Embedding Extraction and Model Inversion
+    description: Attackers may infer or reconstruct original data by probing vector
+      similarity APIs, especially with unrestricted access. This enables model inversion
+      attacks, membership inference, and unauthorized data leakage from stored embeddings.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F01
+          - CCC.Vector.F03
+          - CCC.Vector.F06
+    external-mappings:
+      - reference-id: FINOS-AIR
+        identifiers:
+          - AIR-SEC-002
+  - id: CCC.Vector.TH02
+    title: Embedding and Index Poisoning
+    description: Adversaries may insert malicious or adversarial vectors into the index
+      through ingestion endpoints, polluting the dataset and degrading search quality,
+      or subtly steering results toward specific outcomes.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F02
+          - CCC.Vector.F05
+          - CCC.Vector.F07
+    external-mappings:
+      - reference-id: FINOS-AIR
+        identifiers:
+          - AIR-SEC-009
+  - id: CCC.Vector.TH03
+    title: Cross-modal or Metadata Leakage
+    description: Attackers may infer sensitive information through metadata filters
+      or by correlating embeddings across modalities (e.g., voice and face), bypassing
+      surface-level access controls.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F04
+          - CCC.Vector.F10
+    external-mappings:
+      - reference-id: FINOS-AIR
+        identifiers:
+          - AIR-RC-022
+  - id: CCC.Vector.TH04
+    title: Denial of Service via Query or Ingestion Flooding
```

Comment:
This one overlaps with `CCC.Core.TH12 Resource Constraints are Exhausted`

Reply:
good spot! 

Don't treat anything in this PR as golden @gabriellavengeo  - feel free to change what you will.  The list of core threats has expanded since I wrote this, and in any case it's very much a first draft.